theano/configdefaults.py

```diff
@@ -446,8 +446,12 @@ def safe_no_home(home):
     "The number of stack to trace. -1 mean all.",
     # We default to 6 to be able to know where v1 + v2 is created in the
     # user script. The bigger this number is, the more run time it takes.
-    # We need to default to 7 to support theano.tensor.tensor(...).
-    IntParam(7),
+    # We need to default to 8 to support theano.tensor.tensor(...).
+    # import theano, numpy
+    # X = theano.tensor.matrix()
+    # y = X.reshape((5,3,1))
+    # assert y.tag.trace
```

Comment:
To give the case where 8 is used.

On Tue, Oct 20, 2015 at 4:55 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/configdefaults.py
> <https://github.com/Theano/Theano/pull/3536#discussion_r42552824>:
>
> > @@ -446,8 +446,12 @@ def safe_no_home(home):
> >      "The number of stack to trace. -1 mean all.",
> >      # We default to 6 to be able to know where v1 + v2 is created in the
> >      # user script. The bigger this number is, the more run time it takes.
> > -    # We need to default to 7 to support theano.tensor.tensor(...).
> > -    IntParam(7),
> > +    # We need to default to 8 to support theano.tensor.tensor(...).
> > +    # import theano, numpy
> > +    # X = theano.tensor.matrix()
> > +    # y = X.reshape((5,3,1))
> > +    # assert y.tag.trace
>
> Why is that code in comment here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3536/files#r42552824>.
>